mkosi/__init__.py

```diff
@@ -3789,14 +3789,15 @@ def run_clean(args: Args, config: Config, *, resources: Path) -> None:
 
     if (
         remove_package_cache and
-        config.package_cache_dir and
-        config.package_cache_dir.exists() and
-        any(config.package_cache_dir.iterdir())
+        config.package_cache_dir_or_default().exists() and
+        any(config.package_cache_dir_or_default().iterdir())
```

Comment:
This might be shorted to
```suggestion
        any(config.package_cache_dir_or_default().glob("*"))
```